pkg/osx/req_ext.txt

```diff
@@ -0,0 +1,4 @@
+cryptography==1.1.2
+pyOpenSSL==0.15.1
+pygit2==0.23.2
+
```

Comment:
What does req_ext mean? We can't distribute gpl libs (pygit2) easily, are these libs included in the installer?